src/test/java/net/datafaker/Issue759Test.java

```diff
@@ -89,8 +82,7 @@ void issue759Test() throws InterruptedException {
                 if (allElementsEqual(iters, iterationsPerThread)) {
                     break;
                 } else {
-                    printIterations(iters);
-                    throw new RuntimeException();
+                    throw new AssertionError("Not all of s%s are not equal to %s".formatted(Arrays.toString(iters), iterationsPerThread));
```

Comment:
Sorry, this error message, it's doing my head in. Not all of %s are not equal to .... 

Does this mean: `Not all of %s are different to %s`, or something else?

Reply:
Ooops, one "s" and one "not" were not needed.

The intended variant was:
> Not all of [1,2,3,4,3,6,3,7] are equal to 3.

But we can simplify it, of course.

Reply:
Ah, that reads a lot better indeed! Thanks for improving this!